provider/provider_nodejs_test.go

```diff
@@ -90,6 +89,8 @@ func test(dir string, opts ...providertest.Option) *providertest.ProviderTest {
 		providertest.WithExtraBaselineDependencies(map[string]string{
 			"aws": "5.42.0", // latest v5
 		}),
+		// TODO: make this stricter. https://github.com/pulumi/pulumi-awsx/issues/1166
```

Comment:
I don't think we need a TODO here; NoReplacements is a decent default. Curious what the team thinks.